eventmesh-sdk-java/src/main/java/org/apache/eventmesh/client/grpc/producer/CloudEventProducer.java

```diff
@@ -57,7 +57,7 @@ public CloudEventProducer(EventMeshGrpcClientConfig clientConfig, PublisherServi
     public Response publish(List<CloudEvent> events) {
         logger.info("BatchPublish message, batch size=" + events.size());
 
-        if (events.size() == 0) {
+        if (events.isEmpty()) {
```

Comment:
suggest use CollectionUtils.isEmpty(events).

Reply:
hello @jonyangx  I have made the changes as per suggested by you. Could you please review it and suggest me any further changes if needed.
Thank you:)